iset.mm

```diff
@@ -146032,7 +146032,7 @@ both being (two-sided) identity elements.  (Contributed by AV,
     ${
       grprinvlem.x $e |- ( ( ph /\ ps ) -> X e. B ) $.
       grprinvlem.e $e |- ( ( ph /\ ps ) -> ( X .+ X ) = X ) $.
```

Comment:
Labels should be grpinvalem (also in set.mm).

Reply:
Hmm, naming consistency on hypotheses is a standard we mostly don't hold ourself to (at least not very strictly).

To be honest I'm a bit undecided about whether to change it tomorrow or just not worry about it.

Reply:
Usually, the labels for  hypotheses should start with the label of the first following theorem, or at least of one theorem of the block.

I can fix this later...

Reply:
Yeah, I don't object to renaming labels that way and have submitted those kinds of changes too sometimes.

But, OK, I'll be decisive and go for the "later" in this particular case.